packages/dashboard/src/components/Tile/helpers/__test__/isChecked.spec.js

```diff
@@ -0,0 +1,23 @@
+import isChecked from '../isChecked';
+
+describe('isChecked', () => {
+    it('should return true when is multiple and name is include in value', () => {
+        expect(isChecked(true, [0, 1, 2], 1)).toBe(true);
```

Comment:
in order to improve readability, we should use only one argument as an object thus we can add names to the parameters like:
```icChecked({
    multiple: true,
    value: [...],
    name: 1,
    ...
})```